openapi/components/schemas/Agency.yaml

```diff
@@ -8,6 +8,12 @@ properties:
     type: string
     description: The full name of the agency.
     example: Department of Transportation
+  order:
```

Comment:
Similar to [agencyBudgets](https://github.com/NYCPlanning/ae-zoning-api/blob/3753599daea0c558fab003d71b38330ee1f529bd/openapi/paths/agency-budgets.yaml), `order` is a property on the path, rather than the schema (The schema being concerned only with fields that directly belong to "agencies"). I think  [`openapi/paths/agencies.yaml`](https://github.com/NYCPlanning/ae-zoning-api/blob/3753599daea0c558fab003d71b38330ee1f529bd/openapi/paths/agencies.yaml) would be a better spot